packages/amplify-graphql-searchable-transformer/src/cdk/create-parameters.ts

```diff
@@ -0,0 +1,132 @@
+import { ResourceConstants } from 'graphql-transformer-common';
+import { CfnParameter, Stack } from '@aws-cdk/core';
+
+export function createParametersStack(stack: Stack): Map<string, CfnParameter> {
+  const {
+    ElasticsearchAccessIAMRoleName,
+    ElasticsearchStreamingLambdaHandlerName,
+    ElasticsearchStreamingLambdaRuntime,
+    ElasticsearchStreamingFunctionName,
+    ElasticsearchStreamingIAMRoleName,
+    ElasticsearchDebugStreamingLambda,
+    ElasticsearchInstanceCount,
+    ElasticsearchInstanceType,
+    ElasticsearchEBSVolumeGB,
+  } = ResourceConstants.PARAMETERS;
+  const map = new Map<string, CfnParameter>();
```

Comment:
You can initialize a `Map` with data so that you don't have to call `set()` a bunch of times like you're doing below.